ublox/Cargo.toml

```diff
@@ -12,12 +12,14 @@ readme = "../README.md"
 default = ["std"]
 std = []
 alloc = []
+use-serde = ["serde"]
```

Comment:
You can just leave this out, I think there will automatically be a `serde` feature for the optional dependency.

Reply:
Hello @reitermarkus,
I confirm, this works